frontend/src/components/TextInputComponent.vue

```diff
@@ -6,9 +6,10 @@
       :maxLength="maxLength"
       :placeholder="placeHolder"
       :disabled="disabled"
-      v-model="inputText"
+      :value="props.input"
```

Comment:
Is this still needed?

Reply:
yes i think so, if i understand it right v-model will replaced with this 
```ts
<input
  :value="inputRef"
  @input="inputRef= $event.target.value"
/>
```
and i think the input is loaded before the reference is set, but however if i remove it, it doesnt work